components/command/command-handler/src/main/java/uk/gov/justice/services/core/aggregate/AggregateService.java

```diff
@@ -26,38 +32,50 @@
     Logger logger;
 
     @Inject
+    SnapshotService snapshotService;
+
+    @Inject
     JsonObjectToObjectConverter jsonObjectToObjectConverter;
 
     private ConcurrentHashMap<String, Class<?>> eventMap = new ConcurrentHashMap<>();
 
+    private ObjectInputStreamStrategy objectInputStreamStrategy = new DefaultObjectInputStreamStrategy();
```

Comment:
I'd say there's no reason to inject it. The default one will be used in production. Only for testing we set a test strategy with a custom classloader.